src/components/CreateArticle.jsx

```diff
@@ -0,0 +1,214 @@
+            if (id) {
+            //
+            } else {
+              onSave();
+              const article = await createArticle(this.state);
+              if (!image) {
+                this.onClose();
+              }
+              if (article.success) {
+                this.setState({
+                  id: article.data.id,
+                  title: article.data.title,
+                  description: article.data.description,
+                  body: article.data.body,
+                  readTime: article.data.readTime,
+                  image: article.data.image
+                });
+                this.props.addFlashMessage({
+                  type: 'success',
+                  text: 'Article saved to draft'
+                });
+              } else {
+                this.props.addFlashMessage({
+                  type: 'error',
+                  text: article.message
+                });
+              }
+            }
+          });
+        } else {
+          this.props.addFlashMessage({
+            type: 'error',
+            text: 'failed to upload image'
+          });
+        }
+      }
+    }
+  }
+
+  onChange = (event) => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+    const scroll = event.target.scrollHeight;
+    event.target.style.height = `${scroll}px`;
+  }
+
+  inputChange = (event) => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  }
+
+  fileChange = (event) => {
+    const file = event.target.files[0];
+    this.setState({
+      [event.target.name]: event.target.files[0]
+    });
+    this.readFile(file);
+  }
+
+  readFile = (file) => {
+    if (file) {
+      // eslint-disable-next-line no-undef
+      const FR = new FileReader();
+
+      FR.addEventListener('load', (event) => {
+        this.setState({
+          image: event.target.result
+        });
+      });
+
+      FR.readAsDataURL(file);
+    }
+  }
+
+  onImageClick = () => {
+    document.getElementById('add-image').click();
+  }
+
+  onClose = () => {
+    this.setState({
+      image: null
+    });
+  }
+
+  render() {
+    return (
+      <div className="container create-box">
+        <FlashMessagesList customAlertClass="article-alert" />
+        <div className="row article-row">
+          <div className="col-lg-12 create-box">
+            <h1>
+              <input
+                id="title"
+                name="title"
+                defaultValue={this.props.cache.title}
+                className="form-control"
+                onChange={this.inputChange}
+                type="text"
+                placeholder="Article title"
+              />
+
+            </h1>
+          </div>
+        </div>
+        <div className="row" id="article-container">
+          <div className="col-lg-11" id="article-column">
+            <div id="img-wraper">
+              <img src={this.state.image} id="image-holder" />
+              {(this.state.image !== null)
```

Comment:
Use `(!this.state.image)` which will check for empty strings `""`, `null`, `undefined`, `false` and the numbers `0` and `NaN`.